plugins/system/shortcut/shortcut.xml

```diff
@@ -88,6 +88,17 @@
 						default="ALT + Q"
 					/>
 				</fieldset>
+				<fieldset
+					name="other_actions"
+					label="PLG_SYSTEM_SHORTCUT_OTHER_ACTIONS_FIELDSET_LABEL"
+					>
+					<field
+						name="searchKey"
+						type="keySelect"
+						label="PLG_SYSTEM_SHORTCUT_SEARCH_LBL"
+						default="S"
```

Comment:
It is a really BAD idea to have a hotkey without a modifier for what I hope are very obvious reasons